src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -447,6 +447,46 @@ public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestP
                 id);
         }
 
+        /// <summary>
+        ///     Merges an update dependency pull request
+        /// </summary>
+        /// <param name="pullRequestUrl">Uri of the update dependency pull request to merge</param>
+        /// <param name="parameters">Settings for merge</param>
+        /// <returns></returns>
+        public async Task MergeDependencyPullRequestAsync(string pullRequestUrl, MergePullRequestParameters parameters)
+        {
+            (string accountName, string projectName, string repoName, int id) = ParsePullRequestUri(pullRequestUrl);
+
+            VssConnection connection = CreateVssConnection(accountName);
+            GitHttpClient client = await connection.GetClientAsync<GitHttpClient>();
+
+            string commitToMerge = parameters.CommitToMerge;
+
+            // If the commit to merge is empty, look it up first.
+            if (string.IsNullOrEmpty(commitToMerge))
+            {
+                var prInfo = await client.GetPullRequestAsync(repoName, id);
+                commitToMerge = prInfo.LastMergeSourceCommit.CommitId;
```

Comment:
Are we taking a different approach for Azdo than we are for github? I don't think taking the last commit is correct?